codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketServerProtocolHandshakeHandler.java

```diff
@@ -110,6 +105,10 @@ public void operationComplete(ChannelFuture future) throws Exception {
         }
     }
 
+    private boolean isNotWebSocketPath(FullHttpRequest req) {
+        return checkStartsWith ? !req.uri().startsWith(websocketPath) : !req.uri().equals(websocketPath);
```

Comment:
@Scottmitch performance improvement was for `WebSocketServerProtocolHandshakeHandler.getWebSocketLocation` method where `host` variable was moved out from concat operation and thus concat operation became available for `OptimizeStringConcat` optimization (less allocations + faster).